### PR TITLE
Rename band references to semantic labels

### PR DESCRIPTION
**Pull request by marisn**
See discussion in #1868

### CODE REVIEW

**Review by wenzeslaus — APPROVED** [review]
Great.

I'm just not sure if you meant to change the temporal modules or not. For example, t.rast.list.html still talks about band references, or is that on purpose?

**Comment by marisn** [comment]
> I'm just not sure if you meant to change the temporal modules or not. For example, t.rast.list.html still talks about band references, or is that on purpose?

Hard to tell if I messed up during rebase (again) or just skipped. Went over and replaced even more locations of band references with semantic label.
Now should only i.bands and g.bands remain. What should be the new name for each of them?

**Comment by wenzeslaus** [comment]
> Now should only i.bands and g.bands remain. What should be the new name for each of them?

During the PSC meeting today we talked about options such as my suggestion:

* i.bands -> r.semantic.label (Assigns semantic label to a raster)
* g.bands -> i.band.library (Register band metadata for a semantic label)

...but there was no final decision made.

File: temporal/t.register/t.register.html [hunk 1]
```diff
@@ -137,16 +137,16 @@ <h3>Support for band references</h3>
 T33UYP_20190331T094039_B09|2019-03-31 09:40:39|S2_9
 </pre></div>
 
-In this case <em>t.register</em> assignes to given raster maps band
-references similarly as
+In this case <em>t.register</em> assigns to given raster maps
+semantic label similarly as
```
Comment (wenzeslaus) [inline temporal/t.register/t.register.html:141]: ...assigns ... _a_ semantic label...

**Review by wenzeslaus — APPROVED** [review]

File: temporal/t.register/t.register.html (hunk 1 above)
Comment (marisn) [reply temporal/t.register/t.register.html]: Thanks.

**Comment by marisn** [comment]
> During the PSC meeting today we talked about options such as my suggestion:
> ...but there was no final decision made.

Then lets do it in a separate step. I'll merge this one and then we can handle renaming (or deactivating compilation) of modules as a separate PR.

**Comment by landam** [comment]
Sorry for being late with review. Let's backport to `releasebranch_8_0`.

**Comment by neteler** [comment]
BTW: Shall we have separate G7 and G8 backport labels?

**Comment by landam** [comment]
I think that it's not needed. Milestone is set to `8.0.0`, so only backport to `releasebranch_8_0` is relevant.

**Comment by neteler** [comment]
> I think that it's not needed. Milestone is set to `8.0.0`, so only backport to `releasebranch_8_0` is relevant.

Backport to  `releasebranch_8_0` done by cherry-picking.

**Pull request merged**